### PR TITLE
Add Misc Transfer Types for Staff

### PR DESCRIPTION
**Pull request by dimak1**
*Issue #:* 
- bcgov/entity#18221

*Description of changes:*
- Add Misc Transfer Types for Staff and preserve QS Transfer Types
- Unit test

Screenshot 2024-04-23 at 08 10 25

Screenshot 2024-04-23 at 08 10 34

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: ppr-ui/src/enums/transferTypes.ts [hunk 1]
```diff
@@ -1,17 +1,61 @@
...
+
+  // Transfers Due to Death
   SURVIVING_JOINT_TENANT = 'TRAND',
   TO_EXECUTOR_UNDER_25K_WILL = 'TRANS_AFFIDAVIT',
   TO_ADMIN_NO_WILL = 'TRANS_ADMIN',
-  TO_EXECUTOR_PROBATE_WILL = 'TRANS_WILL'
+  TO_EXECUTOR_PROBATE_WILL = 'TRANS_WILL',
+
+  // Other Transfers
+  ABAN = 'ABAN',
```
Comment (dimak1) [inline ppr-ui/src/enums/transferTypes.ts:18]: I pulled these from the spreadsheet, but I can also update them to something more meaningful. LMK!

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1843-3irqt8cm.web.app

**Review by cameron-eyds — APPROVED** [review]
Looks good!

**Review by doug-lovett — APPROVED** [review]
Looks great.

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1843-3irqt8cm.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1843-3irqt8cm.web.app

**Pull request merged**